modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -88,7 +88,10 @@
     CONT_QRY_SECURITY_AWARE(21),
 
     /** Long operations dump timeout. */
-    LONG_OPERATIONS_DUMP_TIMEOUT(30);
+    LONG_OPERATIONS_DUMP_TIMEOUT(30),
+
+    /** ContinuousQuery with security subject id support. */
+    IGNITE_MESSAGING_SECURITY_AWARE(31);
```

Comment:
I think a new feature is not necessary, you can use CONT_QRY_SECURITY_AWARE. 
Moreover, that's the point in using features at all ?

Reply:
I think it's confused to use CONT_QRY_SECURITY_AWARE for rolling update the IgniteMessage security feature.
I've fixed JavaDoc.

Reply:
@ascherbakoff  I have deleted the IGNITE_MESSAGING_SECURITY_AWARE feature.